src/OOP.java

```diff
@@ -0,0 +1,15 @@
+public class OOP {
+    public static void main(String[] args) {
+     //   Garnek garnek = new Garnek();
+     //   garnek.high = 3;
+     //   garnek.width = 5;
+     //   garnek.color = "black";
```

Comment:
Please remove the commented code or uncomment it if necessary - we do not leave comments in the code